Require/index.js

```diff
@@ -209,6 +209,6 @@ describe("TNS require", function () {
 
     it("should handle JSON file errors", function () {
         require('./RequireJsonCorruptFile');
-        expect(TNSGetOutput()).toBe("File: 'file:///app/shared/Require/RequireJsonCorruptFile/test.json'. JSON Parse error: Unterminated string");
+        expect(TNSGetOutput()).toBe("There was a corrupt or invalid package.json.");
```

Comment:
We try to keep the original error when parsing JSON files. I wonder how can we make this somehow common. What does the V8 engine throw as an error message?